_data/simple-icons.json

```
@@ -3410,6 +3410,16 @@
             "hex": "BE2323",
             "source": "https://www.ebox.ca"
         },
+        {
+            "title": "Eclipse Adoptium",
+            "hex": "FF1464",
+            "source": "https://adoptium.net",
+            "guidelines": "https://www.eclipse.org/org/artwork/guidelines/adoptium-brand-guidelines.pdf",
```

```suggestion
            "guidelines": "https://www.eclipse.org/legal/logo_guidelines.php"
```
This would be more appropriate as the contributing.md mentions that the link is preferred to be a page and not a PDF.